libcxx/test/benchmarks/containers/map.bench.cpp

```diff
@@ -6,944 +6,20 @@
 //
 //===----------------------------------------------------------------------===//
 
-// UNSUPPORTED: c++03, c++11, c++14
+// UNSUPPORTED: c++03, c++11, c++14, c++17
```

Comment:
Can you explain in the commit message why we're no longer able to test with C++17. 
I don't mind the change, but I like to have the reason documented.